ui/main.qml

```diff
@@ -156,11 +156,14 @@ StatusWindow {
                 startupOnboarding.visible = true
             }
             else if(state === Constants.appState.appLoading) {
-                loader.sourceComponent = appLoadingAnimation
+                loader.sourceComponent = undefined
+                appLoadingAnimation.active = true
                 startupOnboarding.visible = false
             }
             else if(state === Constants.appState.main) {
                 // We set main module to the Global singleton once user is logged in and we move to the main app.
+                appLoadingAnimation.active = localAppSettings && localAppSettings.fakeLoadingScreenEnabled
+                appLoadingAnimation.runningProgressAnimation = localAppSettings && localAppSettings.fakeLoadingScreenEnabled
```

Comment:
These can be bindings in the `appLoadingAnimation` Loader; not need to use imperative code

Reply:
Ah I see... we need a new AppState then :smile: 

```nim
type
  AppState* {.pure.} = enum
    StartupState = 0
    AppLoadingState
    MainAppState
```

j/k

Reply:
After this (https://github.com/status-im/status-desktop/pull/9540/files#diff-9c8786956e1d9cf0581b9f73900904aa6b677d5ad9b06a9aa000d97fffd20e0bR165) lands, you might want to bind it to `Global.appReady`